csharp/extractor/Semmle.Extraction/Extractor/Extractor.cs

```diff
@@ -107,17 +107,14 @@ public static string Version
         {
             get
             {
-                // the resources for git information are always attached to the entry` assembly by our build system
+                // the attribute for the git information are always attached to the entry assembly by our build system
                 var assembly = Assembly.GetEntryAssembly();
-                var describeAllStream = assembly.GetManifestResourceStream("git-ql-describe-all.log");
-                var headSHAStream = assembly.GetManifestResourceStream("git-ql-rev-parse.log");
-                if (describeAllStream == null || headSHAStream == null)
+                var versionString = assembly!.GetCustomAttribute<AssemblyInformationalVersionAttribute>();
```

Comment:
`assembly` is probably never `null` in our case, but using `assembly?.` instead of `assembly!.` would be the more conservative approach to handle nullness.